helm/external-secrets/values.yaml

```diff
@@ -0,0 +1,20 @@
+# environment.type: It can be:
+# CDK value: (Cloud Developer's Kit) development identity platform
+# CDM value: CDM (Cloud Deployment Model) identity cloud platform
+# FIDC value: FIDC (Forgerock Identity Cloud) identity cloud platform
+environment:
+  # Configuration relating to the FR Platform
+  fr_platform:
+    type: FIDC
+    fqdn: iam.dev.forgerock.financial
```

Comment:
The only FIDC environment we have is nightly, so we should make these defaults line up. Either we change the type to CDK and have it point to dev.forgerock.financial, or we keep it as FIDC and point everything at nightly.forgerock.financial.

Doesn't really matter which, although if we put the default to nightly we will be able to spot when our pipelines etc are not supplying overrides because dev deployment will break so this might be a better option?